pkg/downloader/manager.go

```diff
@@ -220,42 +220,58 @@ func (m *Manager) downloadAll(deps []*chartutil.Dependency) error {
 	fmt.Fprintf(m.Out, "Saving %d charts\n", len(deps))
 	var saveError error
 	for _, dep := range deps {
+		var isSubChart bool
 		if strings.HasPrefix(dep.Repository, "file://") {
```

Comment:
isn't this unnecessary now? If `IsLocalSubChart` already does this check, why do we need this? Is there a way this can be optimized better?

Reply:
indeed the `if ` is not necessary, Ill remove it.